lib/browser/tag/tag.js

```diff
@@ -171,10 +173,7 @@ function Tag(impl, conf, innerHTML) {
     // parse layout after init. fn may calculate args for nested custom tags
     parseExpressions(dom, self, expressions)
 
-    // unmount automatically when our parent does
-    if (parent) parent.on('unmount', self.unmount)
```

Comment:
I think we can also start cleaning up the tests, making several folders targeting specific riot directives. Who wants to take the lead on this? I can do it eventually together with the new rollup build in the middle of May but I would prefer someone could already start with it